dallinger_scripts/worker.py

```diff
@@ -60,7 +60,7 @@ def main():
         # Default to log.warn because rq logs extremely verbosely at the info
         # level
         worker.log.info = worker.log.debug
-        worker.work(logging_level=os.environ.get("loglevel_worker", LOG_LEVEL))
+        worker.work(logging_level=int(os.environ.get("loglevel_worker", 2)))
```

Comment:
I'm confused, shouldn't the default log level be 20? See eg https://www.logicmonitor.com/blog/python-logging-levels-explained

Reply:
But in any case, why are you now not using LOG_LEVEL as the default like before? This original logic looked fine

Reply:
To be honest, I don't know why we are referring to the environment variable at all here. I think that the reference to os.environ should be deleted. Config loading already loads optionally from environment variable so we don't need to do that here as well. So just write

worker.work(logging_level=LOG_LEVEL)